plugins/right-sizer/version.txt

```diff
@@ -1 +1 @@
-0.1.0
+0.1.1
```

Comment:
I would suggest `0.2`, unless you want to make sure this change gets shipped out to our cluster without needing the team to update the chart.